src/styles/header.scss

```diff
@@ -0,0 +1,66 @@
+:root {
+  --blue-color: #00acdc;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
```

Comment:
Only `box-sizing` is allowed for `*` selector. Move other to `html` , except `text-transform`, use only in those places - where it required